target/linux/imx6/image/lumi.mk

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
Why would we need a subtarget here?

Reply:
I use subtarget to set custom config for kernel to force using CONFIG_CMDLINE_FORCE with predefined partitions to use overlayfs on nand flash.

Reply:
So we will need a new subtarget for a single device here? Sorry, that's not acceptable for us.

Reply:
Is there any other way to provide a custom kernel config for a device?